apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -94,6 +96,23 @@ export class SendMessageEmail extends SendMessageBase {
     });
 
     if (!integration) {
+      if (overrideSelectedIntegration) {
+        await this.createExecutionDetails.execute(
+          CreateExecutionDetailsCommand.create({
+            ...CreateExecutionDetailsCommand.getDetailsFromJob(command.job),
+            detail: DetailEnum.NO_INTEGRATION_INSTANCE_FOUND,
+            source: ExecutionDetailsSourceEnum.INTERNAL,
+            status: ExecutionDetailsStatusEnum.FAILED,
+            isTest: false,
+            isRetry: false,
+            raw: JSON.stringify({
+              integrationIdentifier: overrideSelectedIntegration,
+            }),
+          })
+        );
+
+        return;
+      }
```

Comment:
Show on activity feed that failed, because of a wrong integrationIdentifier

Reply:
I see your point. But I'm not sure about it being understandable enough if he just made a mistake with the identifier. Also, in general, I feel  SUBSCRIBER_NO_ACTIVE_INTEGRATION doesn't really fit the reason we give the error here. 
But for now, you are right it might be better. I will change it.